api/projects/tests/test_permissions.py

```diff
@@ -371,3 +372,29 @@ def test_regular_user_has_no_destroy_permission(self):
 
         # Then - exception thrown
         assert not result
+
+
+@pytest.mark.django_db
+def test_free_plan_has_only_fixed_projects_permission():
+    organisation = Organisation.objects.create(name="Test organisation")
+
+    user = FFAdminUser.objects.create(email="admin@test.com")
+    user_permission_group = UserPermissionGroup.objects.create(
+        name="Users", organisation=organisation
+    )
+    user_permission_group.users.add(user)
+    user.add_organisation(organisation, OrganisationRole.ADMIN)
+
+    project_permissions = ProjectPermissions()
+
+    mock_view.action = "create"
+    mock_view.detail = False
+    mock_request.data = {"name": "Test", "organisation": organisation.id}
+    mock_request.user = user
+
+    for i in range(MAX_PROJECTS_IN_FREE_PLAN):
+        result = project_permissions.has_permission(mock_request, mock_view)
+        Project.objects.create(name=f"Test project{i}", organisation=organisation)
+        assert result
```

Comment:
It's slightly odd to me that we're asserting the result after creating the project. I think we should change this to something like: 

```suggestion
    for i in range(MAX_PROJECTS_IN_FREE_PLAN):
        assert project_permissions.has_permission(mock_request, mock_view)
        Project.objects.create(name=f"Test project{i}", organisation=organisation)
```